pkgs/top-level/all-packages.nix

```diff
@@ -19014,6 +19014,10 @@ with pkgs;
 
   megam = callPackage ../applications/science/misc/megam { };
 
+  ns-3 = callPackage ../development/libraries/science/networking/ns3 {
+    python=python3;
```

Comment:
Is Python 3 required? If so, make the parameter `python3` instead of `python`. is it compatible with 2 and 3? Then just use `python`.